models/kyberswap/avalanche_c/kyberswap_avalanche_c_trades.sql

```diff
@@ -1,5 +1,5 @@
 {{ config(
-    alias = 'trades',
+    alias = 'avalanche_c_trades',
```

Comment:
We can keep the alias as `trades` as we have defined models for every chain in dbt_project.yml
```suggestion
    alias = 'trades',
```